build/run.js

```diff
@@ -136,7 +136,7 @@ commands.build.rust = async function(argv) {
 
         console.log('Checking the resulting WASM size.')
         let stats = fss.statSync(paths.dist.wasm.mainOptGz)
-        let limit = 4.05
+        let limit = 4.10
```

Comment:
This is a lot of memory, given that the number of removed lines nearly matches number of lines added in this PR. Perhaps this is matter of the macro for creating FRP inputs.

Reply:
I increased it too much, sorry. It should be 4.07. Fixing it.

Reply:
What a relief!